docs/source/support/faq.rst

```diff
@@ -0,0 +1,168 @@
+
+
+How can I check my current staking rewards?
+-------------------------------------------
+
+Run::
+
+    nucypher status stakers
+    >    --provider <your_geth_provider>
+    >    --network cassandra
+    >    --staking-address <your_staker_address>``
+
+It takes two periods to observe rewards - if you deposit in period 0 and start a worker in the same period
+(which calls ``confirmActivity()`` to receive work for the next period (1)), then in period 1 the Ursula
+node does work and will get rewarded in period 2 for the work performed in period 1.
+
+.. note::
+
+    `Restaking <https://docs.nucypher.com/en/latest/architecture/sub_stakes.html#re-staking>`_ is enabled by
+    default, so NU inflation rewards are automatically restaked for you, and will be reflected in
+    the ``Staked`` value of the above command.
+
+
+How can I observe the settings (re-staking, winding down) for my stake?
+-----------------------------------------------------------------------
+
+Run::
+
+    nucypher status stakers
+    >    --provider <your_geth_provider>
+    >    --network cassandra
+    >    --staking-address <your_staker_address>``
+
+
+Can I extend the duration of my existing stake?
+--------------------------------------------------------------
+
+Yes, via the `prolong <https://docs.nucypher.com/en/latest/guides/staking_guide.html#prolong>`_ command.
+
+
+How can I reuse an Ursula that was connected to the previous version of the testnet?
+------------------------------------------------------------------------------------
+
+#. Run ``nucypher ursula destroy`` to destroy the current configuration.
+#. Repeat all of the steps with the new tokens in the `Staking Guide <https://docs.nucypher.com/en/latest/guides/staking_guide.html>`_.
+#. Run ``nucypher ursula init`` per the `Ursula Configuration Guide <https://docs.nucypher.com/en/latest/guides/ursula_configuration_guide.html>`_.
+
+
+What is a fleet state?
+----------------------
+
+A symbol which represents your node's view of the network. It is just a
+graphic checksum, so a minor change in the fleet (e.g., a new node joins, a node disappears, etc.)
+will produce a completely different fleet state symbol. A node can have a
+different fleet state than others, which may indicate that a different number of peers are accessible from
+that node's global position, network configuration, etc..
+
+
+Why do I get `NET::ERR_CERT_INVALID` when loading the Ursula node status page?
+------------------------------------------------------------------------------
+
+The status page uses a self-signed certificate, but browsers don’t like it.
+You can usually proceed to the page anyway. If not, try using a different browser.
+
+
+This all seems too complex for me, can I still participate in some way?
+-----------------------------------------------------------------------
+
+We highly recommend delegating to a staking company rather than doing it yourself, if
```

Comment:
```suggestion
We highly recommend delegating to an experienced staker rather than doing it yourself, if
```

Maybe not 'staking company' since that signals that we favour institutional stakers over smaller but competent operators